tests/fixtures/code/File.php

```diff
@@ -13,7 +13,6 @@
 
 // Throw the exception
 throw $exception;
-
```

Comment:
Change here is due to a recent change to the StyleCI presets. Config needs updating, then this change reverting so the tests pass again.